test/tla/cli-integration-tests.md

```diff
@@ -381,3 +381,106 @@ $ TLA_PATH=./tla-path-tests apalache-mc check ./tla-path-tests/ImportingModule.t
 The outcome is: NoError
 ...
 ```
+
+## configure the check command
+
+### configure default Init and Next
+
+```sh
+$ apalache-mc check Config.tla | sed 's/I@.*//'
+...
+  > Command line option --init is not set. Using Init
+  > Command line option --next is not set. Using Next
+...
+  > Set the initialization predicate to Init
+  > Set the transition predicate to Next
+...
+The outcome is: NoError
+...
+```
+
+### configure an invariant in CLI
+
+```sh
+$ apalache-mc check --inv=Inv Config.tla | sed 's/I@.*//'
+...
+  > Set an invariant to Inv
+...
+The outcome is: NoError
+...
+```
+
+### configure via command-line
+
+```sh
+$ apalache-mc check --init=Init1 --next=Next1 --inv=Inv Config.tla | sed 's/I@.*//'
+...
+  > Set the initialization predicate to Init1
+  > Set the transition predicate to Next1
+  > Set an invariant to Inv
+...
+The outcome is: Error
+...
+```
+
+### configure via TLC config
+
+```sh
+$ apalache-mc check --config=Config1.cfg Config.tla | sed 's/[IEW]@.*//'
+...
+  > Loading TLC configuration from Config1.cfg
+...
+  > Config1.cfg: PROPERTY AwesomeLiveness is ignored. Only INVARIANTS are supported.
+...
+  > Set the initialization predicate to Init1
+  > Set the transition predicate to Next1
+  > Set an invariant to Inv1
+...
+The outcome is: NoError
+...
+```
+
+### configure via TLC config and override it!
+
+```sh
+$ apalache-mc check --config=Config1.cfg --init=Init2 --next=Next2 Config.tla | sed 's/[IEW]@.*//'
+...
+  > Loading TLC configuration from Config1.cfg
+...
+  > Set the initialization predicate to Init2
+  > Set the transition predicate to Next2
+  > Set an invariant to Inv1
+...
+The outcome is: Error
+...
+```
+
+### configure missing property in TLC config 
+
+```sh
+$ apalache-mc check --config=Config3.cfg Config.tla | sed 's/[IE]@.*//'
+...
+  > Loading TLC configuration from Config3.cfg
+...
+Configuration error (see the manual): Operator NoLiveness not found (used as a temporal property)
+...
+EXITCODE: ERROR (99)
```

Comment:
The error code does not propagate when the output is preprocessed with `sed`. I am not sure how one could solve that without having an additional script.

Reply:
Ah yeah. My bad.

I think the cleanest fix -- and one with other benefits -- would be to remove the nondeterministic noise on all the output lines so we don't need to clean it up in all the tests. I'd be in favor of not including the timestamps by default, as I suspect that they are *usually* not useful, and only of interest in some special occasions. But it would also make sense to leave the timestamps in by default and add a flag to disable them (perhaps also configurable by environment variable?).

If you think this change is sensible, I can open issue for it, and would be happy to have it in my backlog.

A hack for the moment, if we **need** to capture the exit codes while filtering is still necessary, might be something like

```sh
out=$(cmd 2>&1); res=$?; sed 's/foo/bar/' <<< out
```

but that's quite gnarly, and not probably worth the effort since apalache reports the exit code in any case :D

Reply:
Yeah. People like to see how long it takes to check an individual step. It may take some time to run the model checker. If you like to change the logging output, have a look at `at.forsyte.apalache.infra.log.LogbackConfigurator`. At some point, I gave up on the XML configuration, as it was too unpredictable. So logging is set up in the code.